bin/common.bash

```diff
@@ -88,7 +88,6 @@ validate_version() {
     ck8s_version=$(yq r "${file}" 'global.ck8sVersion')
     if [[ -z "$ck8s_version" ]]; then
         log_error "ERROR: global.ck8sVersion is not set in ${file}"
-        will_exit="true"
```

Comment:
Should the version variable not be required?

Reply:
Oh, right, it should be changed to `maybe_exit` instead.

Reply:
Yeah, we should probably try to declare more variables as `local` to reduce their scope :slightly_smiling_face: